web/src/app/escalation-policies/PolicyStepForm.js

```diff
@@ -265,6 +265,8 @@ export default class PolicyStepForm extends React.Component {
               required
               type='number'
               mapOnChangeValue={(value) => value.toString()}
+              min={1}
```

Comment:
Same thing: I believe this can be 0, which means that it will cause the step to immediately escalate

Reply:
addressed, changes made as requested.